projects/lionheart/Action.cpp

```diff
@@ -4,125 +4,180 @@
 namespace lionheart
 {
 
-  class MoveImpl:public ActionImpl
+  class MoveImpl : public ActionImpl
   {
-    public:
-      MoveImpl(int d):ActionImpl(),dist(d){}
-      std::unique_ptr<ActionImpl> clone() const override
+  public:
+    MoveImpl(int d) : ActionImpl(), dist(d)
```

Comment:
Your formatting changes create noise that makes it harder to see the actual changes.  Please revert so that I can review intended changes only.